### DISCUSSION TITLE
Eror

### DISCUSSION THREAD

**Discussion by mistercool19** [sherlock-project/sherlock#1083] [Q&A]
How do I uppgrade?
![A149055E-6FBF-44B7-B7C2-AD3A85476675](https://user-images.githubusercontent.com/68437343/119638871-5f759a00-be17-11eb-9026-bc5e24cf6bf9.jpeg)

**Comment by tsmitsmitonline** [comment]
Linux/MacOS: python -m pip install -U pip
Microsoft: py -m pip install -U pip

See: https://pip.pypa.io/en/stable/installing/

**Reply by mistercool19** [reply]
I am using repl.it or cloud shell does this works also?

**Comment by SethFalco** [comment]
That is a warning, not an error.
Also, it tells you exactly how to resolve it already in the message! ^-^'

```sh
pip install --upgrade pip
```

**Comment by Cfomodz** [comment]
Seth answered this, can be marked as answered / closed please :)
